app/services/facilities/website_url_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Facilities
+  class WebsiteUrlService
+    FACILITY_TYPES = {
+      'va_benefits_facility' => 'VBA',
+      'va_health_facility' => 'VHA',
+      'va_cemetery' => 'NCA',
+      'vet_center' => 'VC'
+    }.freeze
+
+    def initialize
+      source = Rails.root.join('lib', 'facilities', 'website_data', 'websites.csv')
+      station_websites = CSV.read(source, headers: true)
+      @websites = map_websites_to_stations(station_websites)
+    end
+
+    def find_for_station(id, type)
+      unique_id = "#{id}_#{FACILITY_TYPES[type]}"
+      entry = @websites[unique_id]
+      entry.nil? ? '' : entry
```

Comment:
It looks like we currently return `"website": null` when there isn't one, not an empty string, so I think we can drop this line and modify the corresponding test.

Reply:
Updated to return nil instead of an empty string!